### PR TITLE
Update clinic-user-guide-for-tiup.md

### PR DESCRIPTION
**Pull request by qqqdan**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [X ] master (the latest development version)
- [X ] v6.4 (TiDB 6.4 versions)
- [X ] v6.3 (TiDB 6.3 versions)
- [X ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: clinic/clinic-user-guide-for-tiup.md [hunk 1]
```diff
@@ -75,7 +75,7 @@ Before using PingCAP Clinic, you need to install Diag (a component to collect da
 
     > **Note:**
     >
-    > - When accessing Clinic Server for the first time, before getting a token, you need to log in to [Clinic Server](https://clinic.pingcap.com.cn) using your AskTUG account and create an organization first.
+    > - When accessing Clinic Server for the first time, before getting a token, you need to refer [Quick Start with PingCAP Clinic](/clinic/quick-start-with-clinic.md) to do the pre-steps.
```
Comment (Oreoxmt) [inline clinic/clinic-user-guide-for-tiup.md:78]:
```suggestion
    > - When accessing Clinic Server for the first time, before getting a token, you need to refer [Quick Start with PingCAP Clinic](/clinic/quick-start-with-clinic.md#Prerequisites) and complete the prerequisites.
```

**Review by Oreoxmt — APPROVED** [review]

File: clinic/clinic-user-guide-for-tiup.md [hunk 2]
```diff
@@ -75,7 +75,7 @@ Before using PingCAP Clinic, you need to install Diag (a component to collect da
 
     > **Note:**
     >
-    > - When accessing Clinic Server for the first time, before getting a token, you need to log in to [Clinic Server](https://clinic.pingcap.com.cn) using your AskTUG account and create an organization first.
+    > - When accessing Clinic Server for the first time, before getting a token, you need to refer [Quick Start with PingCAP Clinic](/clinic/quick-start-with-clinic.md#Prerequisites) and complete the prerequisites.
```
Comment (Oreoxmt) [inline clinic/clinic-user-guide-for-tiup.md:78]:
```suggestion
    > - When accessing Clinic Server for the first time, before getting a token, you need to refer [Quick Start with PingCAP Clinic](/clinic/quick-start-with-clinic.md#prerequisites) and complete the prerequisites.
```

File: clinic/clinic-user-guide-for-tiup.md [hunk 3]
```diff
@@ -75,7 +75,7 @@ Before using PingCAP Clinic, you need to install Diag (a component to collect da
 
     > **Note:**
     >
-    > - When accessing Clinic Server for the first time, before getting a token, you need to log in to [Clinic Server](https://clinic.pingcap.com.cn) using your AskTUG account and create an organization first.
+    > - When accessing Clinic Server for the first time, before getting a token, you need to refer [Quick Start with PingCAP Clinic](/clinic/quick-start-with-clinic.md#prerequisites) and complete the prerequisites.
```
Comment (shichun-0415) [inline clinic/clinic-user-guide-for-tiup.md:78]:
```suggestion
    > - When accessing Clinic Server for the first time, before getting a token, you need to prepare the environment by referring to [Quick Start with PingCAP Clinic](/clinic/quick-start-with-clinic.md#prerequisites).
```

**Review by shichun-0415 — APPROVED** [review]
Rest LGTM

**Pull request merged**